frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/CurlLink/index.tsx

```diff
@@ -29,9 +34,16 @@ const CurlLink: FC<Props> = ({ fileName, handleAnalytics, link }) => {
         </CodeMask>
       </CodeWrapper>
       <Tip>
-        If you prefer not to download this dataset directly in your browser, you
-        can optionally use the provided cURL link to download via the terminal.
-        The above link will be valid for 1 week.
+        This cURL command is valid forever. All datasets on CZ CELLxGENE
```

Comment:
I am still seeing a (not valid forever) presigned url being presented in the download modal in dev; since I don't see changes in this PR to display a (valid forever) Discover API url, it seems that we are jumping the gun on introducing this change in the text.

Am I missing something?